iktomi/web/app.py

```diff
@@ -16,15 +16,17 @@
 ip_number = '(\d|[1-9]\d|1\d{2}|2[0-4]\d|25[0-5])'
 dns_letter = '[a-z\d]([a-z\d\-]*[a-z\d])?'
 port = '(?::\d+)'
-HOSTNAME_REGEX = re.compile("^({letter}\.)*{letter}[a-z]{port}?$".format(letter=dns_letter, 
+HOSTNAME_REGEX = re.compile("^({letter}\.)*{letter}{port}?$".format(letter=dns_letter,
                                                                          port=port), re.I)
-IP_REGEX = re.compile("^({number}\.){times}{number}{port}?$".format(number=ip_number, 
-                                                                    times="{3}", 
+IP_REGEX = re.compile("^({number}\.){times}{number}{port}?$".format(number=ip_number,
```

Comment:
Defining number pattern separately is quite reasonable to me. I'm ok with defining port separately too. But why times? Why not just write `{{3}}` in the first place?